imagetest/fixtures.go

```diff
@@ -119,6 +119,15 @@ func (t *TestVM) Reboot() error {
 		return err
 	}
 
+	stopInstancesAfterRebootStep, err := t.testWorkflow.addStopStep("stoped-"+t.name, t.name)
+	if err != nil {
+		return err
+	}
+
+	if err := t.testWorkflow.wf.AddDependency(stopInstancesAfterRebootStep, waitStartedStep); err != nil {
+		return err
+	}
+
```

Comment:
the result of this new block will be that every vm that has a reboot added will be stopped at the end. i don't think this is correct.